core/src/androidTest/java/org/hisp/dhis/android/core/legendset/ProgramIndicatorLegendSetLinkTableInfoStoreIntegrationShould.java

```diff
@@ -36,10 +36,10 @@
 import org.junit.runner.RunWith;
 
 @RunWith(D2JunitRunner.class)
-public class ProgramIndicatorLegendSetLinkStoreIntegrationShould
+public class ProgramIndicatorLegendSetLinkTableInfoStoreIntegrationShould
```

Comment:
The "TableInfo" word in the name is not necessary. Maybe this was accidentally included as part of a refactor